tests/ResponseTest.php

```diff
@@ -361,4 +361,24 @@ public function test_responsable_with_invalid_key(): void
             $page['props']['resource']
         );
     }
+
+    public function test_the_page_url_is_prefixed_with_the_proxy_prefix(): void
```

Comment:
Note that this test passed prior to the changes. It exists to ensure no regression on the issue that #333 was solving.